products/traefik.md

```diff
@@ -23,87 +23,94 @@ auto:
 # Table with releaseCycles/releaseDate/support: https://doc.traefik.io/traefik/deprecation/releases
 # Estimation: support(x) = releaseDate(x+1)
 releases:
--   releaseCycle: "2.10"
-    releaseDate: 2023-04-24
+-   releaseCycle: "2.11"
+    releaseDate: 2024-02-12
     support: true
     eol: false
+    latest: "2.11.0"
+    latestReleaseDate: 2024-02-12
+
+-   releaseCycle: "2.10"
+    releaseDate: 2023-04-24
+    support: 2024-02-12
```

Comment:
I don't understand the meaning of `support` if it's always the same date as `eol`. Is the `support` column really needed ? Are you sure of this change ?

Reply:
The support column is needed for the following combination, see https://doc.traefik.io/traefik/deprecation/releases/:
> The last minor after releasing a new major will be supported for 1 year following the major release

Until now we don't have this combination :smile:

I try to set a date for eol instead of true/false. From the pattern with the new release cycles, the older become unsupported/eol. If you find it better the old way I can revert the modifications.

Reply:
Nope, make sense except maybe for version 1.7. But I did not found any information about when 1.7 entered security support, so let's leave it like that.